constraint_generation/src/execution_data/executed_template.rs

```diff
@@ -21,14 +21,17 @@ pub struct ExecutedTemplate {
     pub report_name: String,
     pub inputs: SignalCollector,
     pub outputs: SignalCollector,
+    pub ordered_signals: Vec<String>,
```

Comment:
At the moment of executing a template we can compute all these new information that will be later used in the DAG to generate the new r1cs file.